free-3.md

```diff
@@ -10,59 +10,116 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This idea has to do with asynchronous tasks. It's not super well fleshed out,
+but I know it will be something similar to cron, but making it very
+easy to automate tasks.
 
-### Why a language?
-_Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+I've never used cron myself (I would definitely try it out before I try to
+make a new version or anything), but I've heard that it can be a bit
+clunky and not super intuitive to learn.  Hopefully anyone who needs to
```

Comment:
I think cron may already be a DSL.  I've used it and it seems *incredibly* useful and powerful although I agree certain elements of it required learning.  The difficult elements of it I don't know how to make simpler though...